.cicd/scripts/srw_ftest.sh

```diff
@@ -88,8 +88,6 @@ module load wflow_${platform,,}
 conda activate srw_app
 set -e -u
 
-export PYTHONPATH=${workspace}/ush/python_utils/workflow-tools:${workspace}/ush/python_utils/workflow-tools/src
-
```

Comment:
PYTHONPATH shenanigans are no longer needed. In fact, they cause problems when trying to call the command line tools from the conda environment.